config/prod/dian-uat-hm-migration-s3.yaml

```diff
@@ -30,6 +30,7 @@ parameters:
     - "arn:aws:iam::024584448019:user/cleardata/customer/michael.votaw"
     - "arn:aws:sts::055273631518:assumed-role/AWSReservedSSO_Administrator_1087bdcd71db6704/aaron.hayden@sagebase.org"
     - "arn:aws:sts::055273631518:assumed-role/AWSReservedSSO_Viewer-Plus_9fa0058a3048f404/bruce.hoff@sagebase.org"
+    - "arn:aws:sts::055273631518:assumed-role/AWSReservedSSO_Developer_d1a84a78c9777596/bruce.hoff@sagebase.org"
     - "arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruce.hoff@sagebase.org"
```

Comment:
i think this is invalid, this role doesn't exist in account `563295687221`

Reply:
To check, I logged into org-sagebase-sandbox as a Developer and in the upper right menu I see:

```
My account: 563295687221
Federated Login: AWSReservedSSO_Developer_baa6fed639faf5e7/bruce.hoff@sagebase.org
```